packages/react/src/primitives/utils/getTestId.ts

```diff
@@ -0,0 +1,5 @@
+export const getTestId = (
```

Comment:
Should we add some small unit tests? I'm sure this has coverage via component unit tests but just as a best practice